js/core/pubsub.js

```diff
@@ -0,0 +1,27 @@
+export default class PubSub {
+    constructor() {
+        this.events = {};
+    }
+
+
+    subscribe(event, callback) {
+        let self = this;
```

Comment:
```suggestion
        const { events }= this;
```
이렇게 events만 받아와서 사용하는건 어떨까요?

Reply:
한번 시도해보고 문제가 없으면 고쳐도 될것같습니다